util/libcrypto.num

```diff
@@ -4400,3 +4400,5 @@ EVP_PKEY_meth_remove                    4343	1_1_1	EXIST::FUNCTION:
 OPENSSL_sk_reserve                      4344	1_1_1	EXIST::FUNCTION:
 CRYPTO_atomic_read                      4345	1_1_1	EXIST::FUNCTION:
 CRYPTO_atomic_write                     4346	1_1_1	EXIST::FUNCTION:
+DH_rfc7919_create                       4347	1_1_1	EXIST::FUNCTION:DH
+DH_get_rfc7919_nid                      4348	1_1_1	EXIST::FUNCTION:DH
```

Comment:
It seems that in other similar cases subroutine that return DH \* are called DH_get_something. In other words question is what is rationale behind calling this one for DH_rfc7919_create, and not for example DH_get_rfc7919_by_nid...

Reply:
Well it is creating a DH structure and populating it with RFC7919 parameters rather than retrieving it from somewhere else and the choice of name reflects that. I don't really mind too much what it's called. The function `EC_KEY_new_by_curve_name` serves a similar purpose so maybe `DH_new_by_rfc7919_nid`?

Reply:
Effectively I was referring to DH_get_xxx_yyy in dh/dh_rfc5114.c. Actually I don't have real suggestion for alternative, name just appears inconsistent to me. So I suppose the query was rather to encourage to give it an extra consideration. But new_by_nid sounds better...

Reply:
Yes I'll change it to DH_new_by_nid and DH_get_nid. There's no reason these should be confined to RFC7919 parameters: we may add new NIDs in future.